app/components/dns-record/form.tsx

```diff
@@ -0,0 +1,63 @@
+import { AddIcon, InfoIcon } from '@chakra-ui/icons';
+import { Button, Input, Select, Textarea, Tooltip, VStack } from '@chakra-ui/react';
+import { Form } from '@remix-run/react';
+import FormField from './form-field';
+
+interface FormParams {
```

Comment:
`DomainFormProps` would be the more usual name for this

Reply:
I didn't go for `DomainFormProps` cause it should actually be `DnsRecordFormProps` and that felt a bit long. I could change it if the length doesn't matter

Reply:
It should go with your component, so name it `*Props`

Reply:
To be clear, do you want it to be `DnsRecordFormProps` or just `*Props`, or `FormProps` ?

Reply:
Whatever the name of the component actually is, so if it's `Form`, then `FormProps`.  But your component is `DnsRecordForm` right?  So `DnsRecordFormProps`.

Reply:
Alright, thanks for the clarification!